test/Microsoft.Buld.Sql.Tests/BuildTests.cs

```diff
@@ -137,5 +137,22 @@ public void VerifyBuildFailureWithUnresolvedReference()
             // Verify failure
             Assert.AreEqual(1, exitCode, "Build is expected to fail.");
         }
+
+        [Test]
+        public void VerifyBuildWithProjectReference()
+        {
+             this.AddProjectReference("ReferenceProj/ReferenceProj.sqlproj");
+
+             // Since reference proj is in a subdirectory it gets picked up by default globbing pattern, excluding it here
```

Comment:
can you also add a test where the referenced project is in a different directory, not in a subdirectory?

Reply:
Added